### PR TITLE
Update mnist_softmax.py

### PR DESCRIPTION
**Pull request by AfirSraftGarrier**
Move some comments to the right place.

### CODE REVIEW

**Comment by AfirSraftGarrier** [comment]
I signed it!

File: tensorflow/examples/tutorials/mnist/mnist_softmax.py [hunk 1]
```diff
@@ -58,16 +58,16 @@ def main(_):
...
+  # Train
   for _ in range(1000):
     batch_xs, batch_ys = mnist.train.next_batch(100)
     sess.run(train_step, feed_dict={x: batch_xs, y_: batch_ys})
 
-  # Test trained model
-  correct_prediction = tf.equal(tf.argmax(y, 1), tf.argmax(y_, 1))
-  accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))
-  print(sess.run(accuracy, feed_dict={x: mnist.test.images,
+    # Test trained model
```
Comment (caisq) [inline tensorflow/examples/tutorials/mnist/mnist_softmax.py]: I think it is intended to run all the training iterations before testing it.

Comment (AfirSraftGarrier) [inline tensorflow/examples/tutorials/mnist/mnist_softmax.py]: Thx, I just found!

**Comment by caisq** [comment]
@tensorflow-jenkins test this please.

**Review by caisq — APPROVED** [review]

**Pull request merged**